docs/UserGuide.md

```diff
@@ -86,15 +86,87 @@ This marks the end of the quick start. Click
 
 ## 3. About
 
+This section aims to help familiarise you with how our application works as well as the terminologies that we use
+throughout our application and user guide.
+
 ### 3.1 Special Icons
 
+Here are the special icons used throughout the User Guide to help you utilise the User Guide more effectively.
+
+#### Tips
+
+Tips are tidbits of helpful information that can help you have a better experience with DonnaFin.
+
+<div markdown="span" class="alert alert-primary">:bulb:
+Tip: Tips are helpful!
+</div><br>
+
+#### Notes
+
+Notes are here to provide the User with extra information that they may find useful when using DonnaFin.
+
+<div markdown="span" class="alert alert-primary">:information_source:
+Take notes when you see this icon!
+
+
+#### Caution 
+
+Cautions are around to warn users of potential unwanted side effects that may occur on certain parts of DonnaFin if the 
```

Comment:
warn you of potential unwanted side effects that may occur when using certain features of DonnaFin if you fail to follow this guide properly or tamper with the files of the application without adequate knowledge.